src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -209,53 +214,61 @@ public void downloadZippedFile(Environment env, HttpServletRequest request,
 
 			String type = Utils.getJSONString(contentItem, CONTENT_JSON_ATTR_TYPE);
 			String title = Utils.getJSONString(contentItem, CONTENT_JSON_ATTR_TITLE);
-
-			// files
-			if (contentUrl == null && contentUrl.length() == 0) {
+			String copyrightAlert = Utils.getJSONString(contentItem,
+					CONTENT_JSON_ATTR_COPYRIGHT_ALERT);
+			if (BOOLEAN_TRUE.equals(copyrightAlert))
+			{
+				// do not migrate if the item is with copyright
+				zipStatus.append(title + " is not downloaded because of copyright alert. " + LINE_BREAK);
+				error_flag = true;
```

Comment:
It is restriction. If the CTools resource has the copyright alert flag, the tool will not transfer the resource content, and instead indicates "due to copyright alert" inside status attribute.